src/components/FeatureSetsPanel/FeatureSetsPanelSchema/FeatureSetsPanelSchema.js

```diff
@@ -20,7 +20,7 @@ const FeatureSetsPanelSchema = ({
   const handleEntitiesOnBlur = () => {
     const entitiesArray = data.entities
       .split(',')
-      .map(entity => ({ name: entity, value_type: 'str' }))
+      .map(entity => ({ name: entity.trim(), value_type: 'str' }))
```

Comment:
That works.
Another approach is to use: `data.entities.trim().split(/[, ]+/)` to split the string on any sequence of spaces and commas that is longer than 1.
So the string `'   abc         ,     def,     ,ghi '` will be split to `['abc', 'def', 'ghi']`.